### PR TITLE
Migrate entries to workbook by copy

### PR DESCRIPTION
**Pull request by Sergey-weber**
Added new dialog SelectMigrationToWorkbookDialog - for selecting migration entries by copy or transfer
Modify MigrateToWorkbookDialog
Added new handler for migration entries by copy

### CODE REVIEW

**Comment by Sergey-weber** [comment]
Тикет на документаторов - https://st.yandex-team.ru/YCDOCS-12857

File: src/ui/components/SelectMigrationToWorkbookDialog/SelectMigrationToWorkbookDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import {closeDialog, openDialog} from '../../store/actions/dialog';
+import DialogManager from '../DialogManager/DialogManager';
+import {DIALOG_MIGRATE_TO_WORKBOOK} from '../MigrateToWorkbookDialog/MigrateToWorkbookDialog';
+
+import {Body} from './Body/Body';
+import {migrateByCopy, migrateByTransfer} from './constants';
+
+import './SelectMigrationToWorkbookDialog.scss';
+
+const b = block('dl-select-migration-to-workbook-dialog ');
```
Comment (imsitnikov) [inline src/ui/components/SelectMigrationToWorkbookDialog/SelectMigrationToWorkbookDialog.tsx:18]: Remove space plz

File: src/ui/components/MigrateToWorkbookDialog/MigrateToWorkbookDialog.scss [hunk 2]
```diff
@@ -11,19 +11,4 @@
         padding-bottom: var(--yc-dialog-side-padding);
         overflow-y: auto;
     }
-
-    &__error-block {
```
Comment (imsitnikov) [inline src/ui/components/MigrateToWorkbookDialog/MigrateToWorkbookDialog.scss:15]: These styles are needed

File: src/ui/components/MigrateToWorkbookDialog/MigrateToWorkbookDialog.tsx [hunk 3]
```diff
@@ -5,14 +5,22 @@ import {Dialog} from '@gravity-ui/uikit';
 import block from 'bem-cn-lite';
 import {I18n} from 'i18n';
 import {useDispatch, useSelector} from 'react-redux';
+import {ConnectorType} from 'shared';
 import {ErrorContent} from 'ui/index';
+import {getRelations} from 'ui/store/actions/copyEntriesToWorkbook';
```
Comment (imsitnikov) [inline src/ui/components/MigrateToWorkbookDialog/MigrateToWorkbookDialog.tsx:10]: Would be better not to use actions and selectors from another store. The `copyEntriesToWorkbook` store was created for copying entries between workbooks dialog. Can we duplicate `getRelations` and selectors to `migrationToWorkbook` store?

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: src/ui/components/SelectMigrationToWorkbookDialog/constants.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+const migrateByCopy = {
+    title: i18n('title_copy'),
+    label: i18n('label'),
+    description: i18n('description_copy'),
+    icon: <Icon data={CopyArrowRight} size={32} />,
+    list: [
+        {
+            icon: <Icon data={Magnifier} />,
+            text: i18n('list_first_item_copy'),
```
[the remark this answers is not in the document]
Comment (Sergey-weber) [reply src/ui/components/SelectMigrationToWorkbookDialog/constants.tsx]: realised key storage in constants, and in the component getting texts using these keys

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**